From cb4ca128b0867d9cc92f22501430d0775018f5f1 Mon Sep 17 00:00:00 2001
From: Flopster101 <nahuelgomez329@gmail.com>
Date: Thu, 18 Jun 2026 15:39:31 -0300
Subject: [PATCH] s5e8825-common: shims: Fix epicoperator shimmed function name

Backport the exact one-line upstream fix to the pinned LineageOS 23.0 source
graph. The FYH2 libexynoscamera3 blob imports createScenarioOperator, not
createOperator.

Upstream-Commit: cb4ca128b0867d9cc92f22501430d0775018f5f1
Signed-off-by: Flopster101 <nahuelgomez329@gmail.com>
---
 shims/libepicoperator/epicoperator.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shims/libepicoperator/epicoperator.c b/shims/libepicoperator/epicoperator.c
index 448baab..7fbefed 100644
--- a/shims/libepicoperator/epicoperator.c
+++ b/shims/libepicoperator/epicoperator.c
@@ -1,3 +1,3 @@
-unsigned int createOperator(int param_1 __attribute__((unused))) {
+unsigned int createScenarioOperator(int param_1 __attribute__((unused))) {
     return 0;
 }
-- 
2.51.0

